assets/src/main/java/bisq/asset/coins/Webchain.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of Bisq.
+ *
+ * Bisq is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or (at
+ * your option) any later version.
+ *
+ * Bisq is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public
+ * License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with Bisq. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package bisq.asset.coins;
+
+import bisq.asset.Coin;
+import bisq.asset.RegexAddressValidator;
+
+public class Webchain extends Coin {
+
+    public Webchain() {
+        super("Webchain", "WEB", new RegexAddressValidator("^0x[0-9a-fA-F]{40}$"));
+    }
+}
```

Comment:
Add newline at the end of file

Reply:
@blabno i created new PR and added newline at the end of file here: #2167 but seems same changes.